app/views/courses/TeacherClassView.coffee

```diff
@@ -1,7 +1,8 @@
 require('app/styles/courses/teacher-class-view.sass')
 RootView = require 'views/core/RootView'
 State = require 'models/State'
-template = require 'templates/courses/teacher-class-view'
+fullPageTemplate = require 'templates/courses/teacher-class-view-full'
+viewTemplate = require 'templates/courses/teacher-class-view'
```

Comment:
Naming suggestion: `contentTemplate` or `templateWithoutNav`

Reply:
Took a stab at improving name - let me know what you think.